events.go

```diff
@@ -286,26 +286,26 @@ type EventDeploymentFailed struct {
 
 // EventDeploymentInfo describes a 'deployment_info' event.
 type EventDeploymentInfo struct {
-	EventType   string          `json:"eventType"`
-	CurrentStep *DeploymentStep `json:"currentStep"`
-	Timestamp   string          `json:"timestamp"`
-	Plan        *DeploymentPlan `json:"plan"`
+	EventType   string            `json:"eventType"`
+	CurrentStep *StepActions `json:"currentStep"`
+	Timestamp   string            `json:"timestamp"`
+	Plan        *DeploymentPlan   `json:"plan"`
```

Comment:
It looks like `gofmt` didn't run on here and on the other blocks in `events.go`.